src/pages/settings/Wallet/PaymentMethodList.js

```diff
@@ -182,23 +200,22 @@ function PaymentMethodList({
     fundList,
     filterType,
     isLoadingPaymentMethods,
-    listHeaderComponent,
-    network,
-    onListContentSizeChange,
     onPress,
-    shouldEnableScroll,
     shouldShowSelectedState,
     shouldShowAddPaymentMethodButton,
     shouldShowAddBankAccount,
     shouldShowEmptyListMessage,
     shouldShowAssignedCards,
     selectedMethodID,
+    listHeaderComponent,
+    onListContentSizeChange,
+    shouldEnableScroll,
     style,
-    translate,
 }) {
-    const filteredPaymentMethods = useMemo(() => {
-        const paymentCardList = fundList || {};
+    const {translate} = useLocalize();
+    const {isOffline: isNetworkOffline} = useNetwork();
```

Comment:
Let's get rid of this destructuring alias and instead change the other instance of `isOffline` to `isFormOffline`